browser/ui/views/frame/vertical_tab_strip_region_view.cc

```diff
@@ -400,6 +400,7 @@ void VerticalTabStripRegionView::OnMouseEntered(const ui::MouseEvent& event) {
 
 void VerticalTabStripRegionView::UpdateNewTabButtonVisibility() {
   bool overflowed =
+      tabs::features::ShouldShowVerticalTabs(browser_) &&
```

Comment:
I think this method could just early return if vertical tab is visible instead of touching `region_view_->new_tab_button()`'s visibility. If not it could be visible in some condition?

Reply:
Yes, we use the original new tab button unless scroll view is overflowed.

Reply:
Got it!